src/c_kzg_4844.c

```diff
@@ -1344,32 +1347,26 @@ static C_KZG_RET compute_r_powers(
  *
  * @remark This function only works for `n > 0`.
  *
- * @param[out] ok                          `true` if the proofs are valid,
- *                                         `false` if not
- * @param[in]  commitments_g1              Array of commitments to verify
- * @param[in]  evaluation_challenges_fr    Array of evaluation of points for the
- *                                         KZG proofs
- * @param[in]  ys_fr                       Array of evaluation results for the
- *                                         KZG proofs
- * @param[in]  proofs_g1                   Array of proofs used for verification
- * @param[in]  n                           The number of
- *                                         blobs/commitments/proofs in the
- *                                         arrays
- * @param[in]  s                           The settings struct containing the
- *                                         commitment verification key (i.e. the
- *                                         trusted setup)
+ * @param[out] ok             True if the proofs are valid, otherwise false
+ * @param[in]  commitments_g1 Array of commitments to verify
+ * @param[in]  zs_fr          Array of evaluation of points for the KZG proofs
+ * @param[in]  ys_fr          Array of evaluation results for the KZG proofs
+ * @param[in]  proofs_g1      Array of proofs used for verification
+ * @param[in]  n              The number of blobs/commitments/proofs
+ * @param[in]  s              The trusted setup
  */
 static C_KZG_RET verify_kzg_proof_batch(
     bool *ok,
     const g1_t *commitments_g1,
-    const fr_t *evaluation_challenges_fr,
+    const fr_t *zs_fr,
     const fr_t *ys_fr,
     const g1_t *proofs_g1,
     size_t n,
     const KZGSettings *s
 ) {
     C_KZG_RET ret;
```

Comment:
How about we are more explicit about `ret` being set in this function and starting with a fail-closed value in all verification functions?

```suggestion
    C_KZG_RET ret = C_KZG_ERROR;
```

Reply:
I don't think that's really necessary. That value will be overwritten every time.